### DISCUSSION TITLE
Livewire Pagination broken with Laravel 9.3.8 update ?

### DISCUSSION THREAD

**Discussion by dominiquevilain** [livewire/livewire#5323] [7. Bugs (archived)]
Hello Livewire Community !

I’m not sure if this is a bug or if i’m doing something wrong, but here is the problem i‘m dealing with. 

I have a HTML table that I use to display 500 Contacts stored in a database.

Whenever I filter the results set with some term using Livewire, the table gets updated, but the pagination still calculates the total amount of contacts from the initial request. If i try do display the total of records used to build de table with a foreach, i see a value corresponding to the amount of contacts filtered, but in the pagination template, it still uses the initial value :

```php
<tbody>
        {{ $contacts->total() }}
        // 3, if i filter with 'Kat' for instance
        @foreach($contacts as $contact)
            //Display info from the 3 contacts extracted by the filter
        @endforeach
    </tbody>
</table>
{{$contacts->links('livewire::tailwind')}}
// in livewire::tailwind, $paginator->total() matches initial count, 500 --}}
```

Am i missing something ? Please note that in the component, the `resetPage()` method is used to reset the page displayed to 1. 

Thank you !

**Comment by dominiquevilain** [comment]
This is in the component : 
```php

    private function queryContacts()
    {
         return Contact::query()
            ->where('name', 'like', '%'.$this->searchTerm.'%')
            ->orWhere('email', 'like', '%'.$this->searchTerm.'%')
            ->orderBy($this->sortField, $this->sortOrder)
            ->paginate($this->perPage);
    }

    public function render()
    {
        return view('livewire.table', [
            'contacts' => $this->queryContacts(),
        ]);
    }
```

**Comment by dominiquevilain** [comment]
Ok, I’ve just reinstalled a fresh new Laravel (9.3.8) and tried two things : 
- basic pagination of a set of users. Nothing fancy, fetching users from a route and inserting links with Users->links(). Works as expected.
- converting the display logic (a basic list) into a livewire Component using WithPagination, and… broken. 

Suspecting a bug on this one, maybe because of the latest Laravel release which is vastly updating blade ?

**Comment by dominiquevilain** [comment]
For illustration, here is the full code of the light example : 

```php
// Component Class (UsersList.php)
namespace App\Http\Livewire;

use App\Models\User;
use Livewire\Component;
use Livewire\WithPagination;

class UsersList extends Component
{
    use WithPagination;

    public function render()
    {
        return view('livewire.users-list', [
            'users' => User::paginate(3),
        ]);
    }
}

``` 

```php
// Component View (users-list.blade.php)
<ul>
    @foreach($users as $user)
        <li>{{$user->name}}</li>
    @endforeach
</ul>

{{ $users->links() }}
```

```html
// Parent Template (Welcome.blade.php)
    <h1>Users</h1>
    <livewire:users-list/>
```

```php
// Route
Route::get('/', function () {
    return view('welcome');
});

```

**Comment by dominiquevilain** [comment]
I’ve just checked my code on a 9.19 Laravel and it works as expected. So, clearly, Laravel 9.3.8 has broken Livewire pagination.

**Comment by joshhanley** [comment]
@dominiquevilain when I get a chance, I will see if I can replicate the issue 🙂
